core/consensus/aura/src/lib.rs

```diff
@@ -411,16 +414,16 @@ fn check_header<B: Block>(slot_now: u64, mut header: B::Header, hash: B::Hash, a
 		// check the signature is valid under the expected authority and
 		// chain state.
 
-		let expected_author = match slot_author(slot_num, &authorities) {
+		let expected_author: AuthorityId<P> = match slot_author::<P>(slot_num, &authorities) {
```

Comment:
`: AuthorityId<P> ` is unecessary no ?

Reply:
`::<P>` should be unnecessary as well. P is bound by `authorities`. There are several of these unnecessary explicit generic parama. Please remove all of them.